05-GitBest.Rmd

```diff
@@ -15,53 +15,58 @@ You can host multiple projects on GitHub at the same time but it would be a mist
 
 -   Never merge someone else's pull request without their consent.
 
--   Branches should be short-lived
+-   Branches should be short-lived.
 
-## Create a meaningful .gitignore file {#Best5.3}
+## Create a meaningful `.gitignore` file {#Best5.3}
 
-.gitignore files are plain text files that instruct the program on predefined files that should be ignored.
+`.gitignore` files are plain text files that instruct the program on predefined files that should be ignored.
 
-Specifically, Github has a file size limit. It is therefore best suited to storing your R scripts and other related files (e.g., ReadMe), **but not raw data and large output files**. These files should be save elsewhere on your computer and *backed up* on your cloud storage.
+Specifically, GitHub has a file size limit. It is therefore best suited to storing your R scripts and other related files (e.g., `README`), **but not raw data and large output files**. These files should be save elsewhere on your computer and *backed up* on your cloud storage. This also allows you to share scripts publically, but to keep potentially sensitive data separate.
 
-Recall that during the creation of your first `repo` we selected the R .gitignore file template, which will be saved in your root folder. You can open this file in RStudio and add or remove any file types.
+Recall that during the creation of your first `repo` we selected the R `.gitignore` file template, which will be saved in your root folder. You can open this file in RStudio and add or remove any file types.
 
 For example:
 
-.pfd .csv .xlsx
+`.pdf` `.csv` `.xlsx`
 
 ## Define the code owner {#Best5.4}
 
-It is important that you give yourself credit for the code you create, and likewise give others credit for the code they created. If you borrow or modified code from another source, this is OK! Just make sure you give credit where credit is due. Similarly, if you are collaborating on code development, define the roles of each author. This can be done at the front end of your scripts and interspersed within the script.
+It is important that you give yourself credit for the code you create, and likewise give others credit for the code they created. If you borrow or modify code from another source, this is OK! Just make sure you give credit where credit is due. Similarly, if you are collaborating on code development, define the roles of each author. This can be done at the front end of your scripts and interspersed within the script.
 
 GitHub has a built-in code owners feature that allows you to grant access to a particular group of staff or partners. Only verified code owners will be able to edit or review the project. The code owner feature can be used for more than assigning individuals to work on the repository. It can automatically assign teams and grant them permission to work on the project and maintain the repository. Birds Canada will be using the team and code owner feature to manage scripts that are shared. See Chapter 5.
```

Comment:
See Chapter 5? Do you mean a different Chapter? Should also link to where you want them to look